### PR TITLE
Enable new multibranch pipeline with a Jenkinsfile

### PR DESCRIPTION
**Pull request by hsablonniere**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
...
+```
+
+### On GNU/Linux
+
+#### Debian/Ubuntu (.deb)
+
+If you are using a GNU/Linux distribution that uses `.deb` packages like Debian or Ubuntu, you can run:
+
+```sh
+apt-get install apt-transport-https gnupg
```
Comment (urcadox) [inline README.md]: This is most likely installed by default

File: README.md [hunk 2]
```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
...
-    . /usr/local/etc/bash_completion
+* The `.rpm` packages are hosted on Bintray.
+* If you want access to the beta channel, you can omit the second line which contains an exclude option.
 
-    # In ~/.zshrc
-    fpath=(/usr/local/share/zsh-completions $fpath)
+#### Arch Linux
 
-### Windows
+If you are using Arch Linux, the packages can be built from AUR with this repo: [clever-tools-bin](https://aur.archlinux.org/packages/clever-tools-bin/).
```
Comment (urcadox) [inline README.md]: I would replace build with installed given that it just downloads a binary.

File: README.md [hunk 3]
```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
...
+If you don't know how to use this, you can run:
 
-Clever Tools is packaged using [chocolatey](https://chocolatey.org):
+```sh
+git clone https://aur.archlinux.org/clever-tools-bin.git clever-tools
+cd clever-tools
+makepkg -s
+pacman -U clever-tools-bin-*.pkg.tar.xz
```
Comment (urcadox) [inline README.md]: You can remove this and put `makepkg -si` instead of just `-s`

File: README.md [hunk 4]
```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
...
-    
-#### CentOS/Fedora/RHEL
+WARNING: ⚠ Before version 1.0.0, you will also need to copy `nodegit.node` file from the archive into your `PATH`.
 
-You need to add our rpm repository which is provided by bintray.
+NOTES:
 
-    curl https://bintray.com/clevercloud/rpm/rpm > /etc/yum.repos.d/bintray-clevercloud-rpm.repo
-    yum install clever-tools
+* The packages are available on Clever Cloud's cellar bucket: [clever-tools-latest_linux.tar.gz](https://clever-tools.cellar.services.clever-cloud.com/releases/latest/clever-tools-latest_linux.tar.gz).
```
Comment (urcadox) [inline README.md]: s/cellar/Cellar/

File: README.md [hunk 5]
```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
...
+curl https://clever-tools.cellar.services.clever-cloud.com/releases/latest/clever-tools-latest_macos.tar.gz > clever-tools.tar.gz
+tar xvzf clever-tools.tar.gz
+cp clever ~/.local/bin/
+```
+
+WARNING: ⚠ Before version 1.0.0, you will also need to copy `nodegit.node` file from the archive into your `PATH`.
+
+NOTES:
+
+* The packages are available on Clever Cloud's cellar bucket: [clever-tools-latest_macos.tar.gz](https://clever-tools.cellar.services.clever-cloud.com/releases/latest/clever-tools-latest_macos.tar.gz).
```
Comment (urcadox) [inline README.md]: s/cellar/Cellar/

File: README.md [hunk 6]
```diff
@@ -6,86 +6,158 @@ Command Line Interface for Clever Cloud.
...
+
+#### Using the `.zip` archive
+
+If you are using Windows but you don't have [chocolatey](https://chocolatey.org) installed, you can download a `.zip` archive and extract the binary in your `PATH`.
+
+WARNING: ⚠ Before version 1.0.0, you will also need to copy `nodegit.node` file from the archive into your `PATH`.
+
+NOTES:
+
+* The packages are available on Clever Cloud's cellar bucket: [clever-tools-latest_win.tar.gz](https://clever-tools.cellar.services.clever-cloud.com/releases/latest/clever-tools-latest_win.zip).
```
Comment (urcadox) [inline README.md]: s/cellar/Cellar/

File: README.md [hunk 7]
```diff
@@ -292,3 +364,4 @@ This job builds a new chocolatey package with the following tasks:
   * This nupkg file artifact is archived on Jenkins.
 
 ⚠️ *The publication of this artifact on the public chocolatey site is still a manual process. You can find more information about why on [this issue](https://github.com/CleverCloud/chocolatey-packages/issues/2).*
+mercredi 29 août 2018, 14:49:21 (UTC+0200)
```
Comment (urcadox) [inline README.md]: wut?

Comment (hsablonniere) [reply README.md]: It's when I do tests :p I'll explain one day ;-)

File: README.md (hunk 1 above)
Comment (hsablonniere) [reply README.md]: I had to do it on the default docker/debian so I was not sure.
I removed it.

File: RELEASE.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+* The `job-*.js` scripts use some external binaries when it was easier (or when there was no strong JS based solution):
+  * `tar`
+  * `zip`
+  * `git`
+  * `npm`
+  * `ssh-keyscan`
+  * `fpm` (which is a ruby gem)
+* We run this Jenkins job on an agent using this Docker image: [clever-tools-builder](https://hub.docker.com/r/clevercloud/clever-tools-builder/).
+* The `Dockerfile` to build this image is here: [docker-runner/Dockerfile](./docker-runner/Dockerfile)
+* The job is triggered on each new commit and tag via a classic Jenkins git hook setup in the GitHub project.
```
Comment (ldoguin) [inline RELEASE.md]: s/tag/tagged/

Comment (hsablonniere) [reply RELEASE.md]: I really meant "on each new commit and each new tag".
I will clarify.

**Pull request merged**